README.md

```diff
@@ -20,4 +21,6 @@ The `deploy-techdocs` action will build your [techdocs](https://brandwatch.dev/d
 
 `your-service-name` is the name of the [backstage](https://github.com/BrandwatchLtd/katalog/tree/main/services) service you want to build a doc for.
 
+`working-directory` let's you optionally set the working directoy where to run the command. The default is the repo root.
```

Comment:
`let's` should be `lets`. I'd also consider changing `where to run the command` to `to run the command in`.

Reply:
@andrewru Thank you!